src/query/storage/cache/redis_cache.go

```diff
@@ -58,9 +59,9 @@ func NewCacheMetrics(scope tally.Scope) CacheMetrics {
 	}
 }
 
-// Update metrcis for a cache hit
+// Update metrics for a cache hit
 func (cm CacheMetrics) CacheMetricsHit(result storage.PromResult) {
-	cm.hitCounter.Inc(1)
+	// cm.hitCounter.Inc(1)
```

Comment:
I think the cache hit/miss rate is a little weird of a metric since, with the sliding window cache, we can get parts of the result from M3DB and parts from Redis so wasn't sure how to define a full hit or full miss in that case, I think the other metrics give a better indicator so I didn't record this

Reply:
if you don't use it, just remove the code, ideally we should never comment out the code in the master branch.